stories/Molecules/Blocks/CountryCard/country-card.scss

```diff
@@ -87,8 +87,8 @@
               &__caption {
                 bottom: 0;
                 opacity: 1;
-                border-left: 1px solid $color-white;
-                border-right: 1px solid $color-white;
+                padding-left: 1px;
+                padding-right: 1px;
                 margin-left: -1px;
```

Comment:
@Shamimkhan321 Lets remove the padding and margin altogether. Use `box-shadow`. Also I do not see `box-sizing` used here? Kindly also add `box-sizing: border-box;` by default on `.pagehero-cards__caption` class.
```
    box-shadow: 0 1px 0 1px $color-white; 
```